### PR TITLE
main/hostapd: security upgrade to 2.7 and wpa_supplicant

### PR DESCRIPTION
**Pull request by andypost**
Ref http://lists.infradead.org/pipermail/hostap/2018-December/039069.html

I will add `wpa_supplicant` to the same PR

### CODE REVIEW

**Comment by andypost** [comment]
@ncopa somehow it needs backport

File: main/wpa_supplicant/APKBUILD [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
+#     - CVE-2017-13077
+#     - CVE-2017-13078
+#     - CVE-2017-13079
+#     - CVE-2017-13080
+#     - CVE-2017-13081
+#     - CVE-2017-13082
+#     - CVE-2017-13086
+#     - CVE-2017-13087
+#     - CVE-2017-13088
+#     - CVE-2018-14526
```
Comment (ncopa) [inline main/wpa_supplicant/APKBUILD]: looks like we fixed CVE-2018-14526 in 2.6-r14 (see line below)

Comment (andypost) [reply main/wpa_supplicant/APKBUILD]: Removed, also fixed in commit message

**Comment by J0WI** [comment]
Is this in the scope of 3.9?

**Pull request closed without merging**